scripts/update-dependencies/project.json

```diff
@@ -0,0 +1,25 @@
+{
+  "version": "1.0.0-*",
+  "description": "Updates the repos dependencies",
+  "compilationOptions": {
+    "emitEntryPoint": true
+  },
+  "dependencies": {
+    "NETStandard.Library": "1.5.0-rc2-23911",
+    "Microsoft.CSharp": "4.0.1-rc2-23911",
+    "System.Runtime.Serialization.Primitives": "4.1.1-rc2-23911",
+    "Microsoft.DotNet.Cli.Build.Framework": "1.0.0-*",
+    "NuGet.Versioning": "3.4.0-rtm-0764",
+    "Newtonsoft.Json": "7.0.1",
+    "Octokit": "0.18.0",
+    "Microsoft.Net.Http": "2.2.29"
+  },
+  "frameworks": {
+    "netstandardapp1.5": {
+      "imports": [
+        "dnxcore50",
+        "portable-net45"
```

Comment:
I think this has to be `portable-net45+win8`. If there's a package that has `portable-net45+[something else]` it is not guaranteed to work in `netstandardapp1.5`. Is there a package that falls into that bucket?